project.clj

```diff
@@ -28,27 +30,34 @@
                          [net.logstash.logback/logstash-logback-encoder "4.7"]
                          [org.codehaus.janino/janino "2.7.8"]
                          [com.fasterxml.jackson.core/jackson-core "2.7.3"]
+                         [org.yaml/snakeyaml "1.13"]
 
+                         [org.apache.maven.wagon/wagon-provider-api "2.10"]
                          [org.apache.commons/commons-exec "1.3"]
+                         [org.apache.httpcomponents/httpclient  "4.5.1"]
```

Comment:
Yeah, I think it was that the jgit dependency in pe-file-sync tries to pull in an older version of the apache http client.